[Bug] Deck Options Crashes on Open

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/2352e419-50e9-425b-8a95-152a4d8ee682

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.ichi2.anki/com.ichi2.anki.DeckOptions}: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter defaultReturnValue
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3898)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4081)
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:91)
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2462)
at android.os.Handler.dispatchMessage(Handler.java:110)
at android.os.Looper.loop(Looper.java:219)
at android.app.ActivityThread.main(ActivityThread.java:8393)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1055)
Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter defaultReturnValue
at com.ichi2.preferences.NumberRangePreference.getPersistedString(Unknown Source:2)
at android.preference.EditTextPreference.onSetInitialValue(EditTextPreference.java:169)
at android.preference.Preference.dispatchSetInitialValue(Preference.java:1624)
at android.preference.Preference.onAttachedToHierarchy(Preference.java:1391)
at android.preference.PreferenceGroup.addPreference(PreferenceGroup.java:171)
at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:113)
at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:49)
at android.preference.GenericInflater.rInflate(GenericInflater.java:497)
at android.preference.GenericInflater.rInflate(GenericInflater.java:502)
at android.preference.GenericInflater.inflate(GenericInflater.java:334)
at android.preference.GenericInflater.inflate(GenericInflater.java:271)
at android.preference.PreferenceManager.inflateFromResource(PreferenceManager.java:343)
at android.preference.PreferenceActivity.addPreferencesFromResource(PreferenceActivity.java:1541)
at com.ichi2.anki.DeckOptions.onCreate(DeckOptions.java:16)
at android.app.Activity.performCreate(Activity.java:8121)
at android.app.Activity.performCreate(Activity.java:8109)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1320)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3871)
... 11 more
```

--- a/AnkiDroid/src/main/java/com/ichi2/preferences/NumberRangePreference.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/preferences/NumberRangePreference.kt
@@ -60,7 +60,7 @@ open class NumberRangePreference : android.preference.EditTextPreference {
      * type. The two methods below intercept the persistence and retrieval methods for Strings and replaces them with
      * their Integer equivalents.
      */
-    override fun getPersistedString(defaultReturnValue: String): String {
+    override fun getPersistedString(defaultReturnValue: String?): String? {
         return getPersistedInt(mMin).toString()
     }
 

